api/repositories.py

```diff
@@ -29,13 +29,14 @@ class UserRepository(Repository):
     def __init__(self, mongo_collection, repository_collection):
         Repository.__init__(self, mongo_collection, repository_collection)
 
-    def update_user_account(self,user_id,name,password,house_name,house_location):
+    def update_user_account(self,user_id,name,password_hash,house_name,house_location):
         user = self.get_user_by_id(user_id)
         if user is None:
-            return None
+            return False
         self.collection.update_one({'user_id': user_id, 'name': name,
-                                    'password': password, 'house_name': house_name,
+                                    'password_hash': password_hash, 'house_name': house_name,
```

Comment:
So you're assuming that this method is called with the password already hashed? Isn't that a bit weird? Wouldn't it be better to hash the password before passing it to the update query?